json/src/test/java/com/basistech/rosette/dm/json/plain/JsonTest.java

```diff
@@ -491,7 +502,16 @@ public void oneWithEverything() {
         crListBuilder.add(topicResult);
         builder.topicResults(crListBuilder.build());
 
+        Embeddings.Builder embeddingsBuilder = new Embeddings.Builder();
+        EmbeddingCollection.Builder embeddingCollectionBuilder = new EmbeddingCollection.Builder();
+        embeddingCollectionBuilder.put(0, new float[]{0, 1});
+        embeddingsBuilder.put(Embeddings.TEXT, embeddingCollectionBuilder.build());
+        embeddings = embeddingsBuilder.build();
+        builder.embeddings(embeddingsBuilder.build());
+
```

Comment:
I'm not ready for a convenience accessor yet.  But I am pointing out the weirdness of getEmbeddings at two levels.  If we can't think of a better name, let's go with it.